src/DebugBar/Storage/RedisStorage.php

```diff
@@ -15,15 +15,15 @@
  */
 class RedisStorage implements StorageInterface
 {
+    /** @var \Predis\Client|\Redis */
     protected $redis;
 
-    protected $hash;
+    protected string $hash;
 
     /**
-     * @param  \Predis\Client $redis Redis Client
-     * @param  string $hash
+     * @param  \Predis\Client|\Redis $redis Redis Client
      */
-    public function __construct($redis, $hash = 'phpdebugbar')
+    public function __construct($redis, string $hash = 'phpdebugbar')
```

Comment:
major breaking change. Narrowing of type of  parameter of public function.

Reply:
Done.